### DISCUSSION TITLE
how to pause video programtically

### DISCUSSION THREAD

**Discussion by ryanleecode** [muxinc/next-video#269] [Q&A] [answered]
For example when it goes out of view, how to pause?

**Comment by luwes** [comment, marked as answer]
I updated the example here with using the paused prop
https://codesandbox.io/p/devbox/next-video-mux-j9nn73

the default player is documented more here
https://docs.mux.com/guides/mux-player-web
https://github.com/muxinc/elements/tree/main/packages/mux-player-react
